src/phylum/ci/ci_base.py

```diff
@@ -133,36 +142,41 @@ def post_output(self) -> None:
     def init_cli(self) -> None:
         """Check for an existing Phylum CLI install, install it if needed, and set the path class instance variable."""
         specified_version = self.args.version
+        # fmt: off
+        install_args = [
+            "--phylum-release", specified_version,
+            "--target", self.args.target,
+            "--phylum-token", self.args.token,
+        ]
+        # fmt: on
         cli_path, cli_version = get_phylum_bin_path(version=specified_version)
         if cli_path is None:
             print(f" [+] Existing Phylum CLI instance not found. Installing version `{specified_version}` ...")
-            install_args = ["--phylum-release", specified_version, "--phylum-token", self.args.token]
             phylum_init(install_args)
         else:
             print(f" [+] Existing Phylum CLI instance found: {cli_version} at {cli_path}")
             if cli_version != specified_version:
                 print(f" [+] Existing version {cli_version} does not match the specified version {specified_version}")
                 if self.args.force_install:
                     print(f" [*] Installing Phylum CLI version {specified_version} ...")
-                    install_args = ["--phylum-release", specified_version, "--phylum-token", self.args.token]
                     phylum_init(install_args)
                 else:
                     print(" [+] Attempting to use existing version ...")
-                    if Version("v3.2.0") >= Version(str(cli_version)):
+                    if Version(str(cli_version)) < Version("v3.2.0"):
                         raise SystemExit(" [!] The existing CLI version must be greater than v3.2.0")
-
-                    # TODO: Account for the case that the existing version does not already have a token in place. This
-                    #       could be the case if the Docker image is built with a known good/compatible CLI version but
-                    #       without a token so that any user can use the image with their own supplied token.
-                    #       https://github.com/phylum-dev/phylum-ci/issues/32
-
                     print(" [+] Version checks succeeded. Using existing version.")
 
         cli_path, cli_version = get_phylum_bin_path(version=specified_version)
         print(f" [+] Using Phylum CLI instance: {cli_version} at {str(cli_path)}")
 
         self._cli_path = cli_path
 
+        # Exit condition: a Phylum API key should be in place or available at this point.
+        # Ensure `capture_output` is used for the subprocess, to keep the token from being printed in (CI log) output
```

Comment:
Seems like `stdout=subprocess.DEVNULL` would be better if that is the purpose.

Reply:
That is much better...forgot about that option...thanks!